README.md

```diff
@@ -405,6 +405,37 @@ curl -HX-Okapi-Tenant:$OKAPI_TENANT -HContent-Type:application/json \
   -XPUT $OKAPI_URL/reservoir/config/oai -d'{"transformer":"marc-transformer::transform"}'
 ```
 
+## Ingest via multipart/form-data
+
+Yet another ingesting alternative is importing via HTTP multipart/form-data at endpoint `/reservoir/upload`.
+
+The named form element `records` is recognized; other names are ignored.
+Remaining configuration is triggered by headers (such as `X-Okapi-Token`) and query parameters.
+
+Currently, two formats are supported.
+
+ * MARC/ISO2709 concatenated records, triggered by Content-Type `application/octet-stream`.
+ * MARCXML collection, triggered by Content-Type `application/xml` or `text/xml`.
+
+The following query parameters are recognized:
+
+ * `sourceId`: required parameter for specifying the source identifier.
+ * `sourceVersion` : optional parameter for specifying source version
+    (default is 1)
+ * `localIdPath` : optional parameter for specifying where to fetch local identifier
+    (default is `$.marc.fields[*].001`)
+
+These query parameters are for debugging and performance testing only:
+
+ * `ingest` optional boolean parameter to determine whether ingesting is to take place (default `true`)
+ * `raw` optional boolean parameter to determine whether to just pipe the stream (default `false`)
+
+For example to ingest a set of MARCXML records via curl from sourceId `BIB1`:
+
+```
+  curl -HX-Okapi-Tenant:$OKAPI_TENANT -Frecords=records100k.xml $OKAPI_URL/reservoir/upload?sourceId=BIB1
```

Comment:
@adamdickmeiss I would expect this to be:
`curl -HX-Okapi-Tenant:$OKAPI_TENANT -F"records=@records100k.xml;type=application/xml" $OKAPI_URL/reservoir/upload?sourceId=BIB1`. Have you tested this?

Reply:
Yes

Reply:
Oops. missing `@`.